dev/devicelab/lib/framework/ios.dart

```diff
@@ -102,6 +104,40 @@ Future<bool> containsBitcode(String pathToBinary) async {
   return !emptyBitcodeMarkerFound;
 }
 
+Future<bool> bonjourServiceFound(String appBundlePath) async =>
```

Comment:
Can we call this dartObservatoryBonjourServiceFound?  or take in `_dartobservatory._tcp` as a parameter?  From the callsite it just looks like we are throwing if any bonjour service is found.